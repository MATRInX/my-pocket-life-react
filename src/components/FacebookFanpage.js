import React from 'react';

const FacebookFanpage = () => (
  <div>FacebookFanpage content</div>
);

export default FacebookFanpage;