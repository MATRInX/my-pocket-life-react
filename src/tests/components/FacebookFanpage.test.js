import React from 'react';
import { shallow } from 'enzyme';
import FacebookFanpage from '../../components/FacebookFanpage';

test('should correctly render FacebookFanpage widget on sidebar', () => {
  const wrapper = shallow(<FacebookFanpage />);
  expect(wrapper).toMatchSnapshot();
});