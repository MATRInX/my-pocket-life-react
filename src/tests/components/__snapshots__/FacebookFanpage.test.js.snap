// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render FacebookFanpage widget on sidebar 1`] = `
<div>
  <Facebook
    cookie={false}
    domain="connect.facebook.net"
    frictionlessRequests={false}
    language="en_US"
    status={false}
    version="v2.9"
    wait={false}
    xfbml={false}
  >
    <Page
      href="https://www.facebook.com/MyPocketLife"
      tabs="timeline"
    />
  </Facebook>
</div>
`;
